package.json

```diff
@@ -55,12 +55,14 @@
     "@commitlint/config-conventional": "^7.0.0",
     "all-contributors-cli": "^5.4.1",
     "ava": "1.0.0-beta.4",
+    "axe-core": "^3.2.2",
     "babel-core": "6.24.1",
     "babel-eslint": "8.0.0",
     "babel-loader": "7.1.2",
     "babel-plugin-istanbul": "4.1.3",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-react-remove-prop-types": "0.4.13",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Do we need this? If yes, won't we need it for client-side as well? This will have a huge impact on bundle-size if we need to bundle it. We can list it as a peer dep or one of the things one needs to do to get keyboard navigation working but it'd be sweet if we can avoid requiring any polyfills.

Reply:
This is "only" a dev-dependency to run async await in tests.
Won't affect bundle size.

Reply:
Well, in that case, have you tried https://babeljs.io/docs/en/babel-plugin-transform-runtime/?

Reply:
Tried now, got  Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.

Reply:
Only seems compatible with 7 of babel:
https://www.npmjs.com/package/@babel/plugin-transform-runtime

Do you want me to upgrade babel-version? It set @ 6.24.1

Reply:
Switched to 
```json
["transform-runtime", { "polyfill": false, "regenerator": true }]
```

Reply:
Umm, yeah there's something for babel 6 as well. Glad you found it.